.github/workflows/ci.yml

```diff
@@ -159,3 +159,34 @@ jobs:
         with:
           directory: coverage
           #files: lcov.info
+
+  doc:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        rust: [nightly]
```

Comment:
Does this make sense for docs?  E.g. in symbolicator we check the doc comments simply on `runs-on: ubuntu-latest` and with stable rust

Reply:
For these specific tests we can do it without, as long as `--cfg doc_cfg` isn't replicated from `docs.rs` setup.

Reply:
I'm not sure what you're saying here, you are adding `--cfg doc_cfg` below?  Is it preferable to check the docs on all platforms and with nightly?

I'm just worried we're blowing up the amount of CI resources for no real benefit with this matrix.

Reply:
1. `nightly`: This is needed to enable `--cfg doc_cfg` for `#![cfg_attr(doc_cfg, feature(doc_cfg))]`. It doesn't seem worth it, since `rustdoc` doesn't validate `doc(cfg())` features to actually exist :(;
2. Having 3 `os`'es in a matrix is not needed, there are only very few os `cfg`s that have no effect on documentation. I'll remove them.

Reply:
Sorry for all the noise (repeated pushes) in all PRs, great lack of focus for the simplest of things :weary:

Reply:
Oh, I forgot that `--cfg doc_cfg` needs nightly.  thanks for reminding me :)